spec/factories/users.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :user do
+    first_name "Richard"
+    last_name  "Mejia"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.